lib/matplotlib/font_manager.py

```diff
@@ -196,11 +196,14 @@ def list_fonts(directory, extensions):
     recursively under the directory.
     """
     extensions = ["." + ext for ext in extensions]
-    return [os.path.join(dirpath, filename)
-            # os.walk ignores access errors, unlike Path.glob.
-            for dirpath, _, filenames in os.walk(directory)
-            for filename in filenames
-            if Path(filename).suffix.lower() in extensions]
+    if sys.platform == 'win32' and directory == win32FontDirectory():
+        return [os.path.join(directory, filename) for filename in os.listdir(directory) if os.path.isfile(filename)]
```

Comment:
I am very confused why this is not failing the linter as too long of a line.....

Reply:
> I am very confused why this is not failing the linter as too long of a line.....

I'm running flake8 before each commit I make, so I think it's just ignored

Reply:
Oh, wrote previous comment before you replied to me :) 
So should I fix it or not?